posts/rails_memory.md

```diff
@@ -147,6 +173,8 @@ The server memory was often below 30% and I satisfied with that results.
 
 # More thinkings
 
+<!-- part 12 -->
+
 You can find docker Ruby with jemalloc images [here](https://hub.docker.com/r/swipesense/ruby-jemalloc)
 It's quite convenient to start with the right choice of memory management library and it will mitigate your pain when discover the memory problems.
```

Comment:
-> It's **recommended** to start with **a** right choice of memory management library **as** it will mitigate your pain **of encountering** memory problems.

Reply:
https://github.com/HoangNguyen679/aphelios/pull/1/commits/43385a881b5b9231888d65f8cdb96da739d3d352 fixed.